sdk/dotnet/Pulumi/Deployment/Deployment_Prepare.cs

```diff
@@ -89,6 +111,7 @@ public partial class Deployment
                 serializedProps,
                 parentURN ?? "",
                 providerRef ?? "",
+                providerRefs ?? new Dictionary<string, string>(),
```

Comment:
If we're going to end up creating a dictionary anyway here, should we just new it up when it's declared?